python/example.py

```diff
@@ -76,13 +76,16 @@ def parse_arguments():
       required=False)
     parser.add_argument('-tls', '--tls', dest='tls', help='Enable encrypted connection',
       required=False, default=False, action='store_true')
+    parser.add_argument('-disableServerVerification', '--disableServerVerification', dest='disableServerVerification',
+                        help='Disable server verification',
```

Comment:
Should this be "Disable TLS server verification." ?

Reply:
Fixed